src/database/hooks/useSaveAnonChatHook.ts

```diff
@@ -15,7 +15,7 @@ import {
 import {getChannelListInfo} from '../../utils/string/StringUtils';
 
 const useSaveAnonChatHook = () => {
-  const {localDb, refresh} = useLocalDatabaseHook();
+  const {localDb, refresh, refreshWithId} = useLocalDatabaseHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `refreshWithId` from `useLocalDatabaseHook()` seems to be a good change. It allows for more specific refreshing of data, which can lead to performance improvements by reducing unnecessary data refreshes.
```diff
-  const {localDb, refresh} = useLocalDatabaseHook();
+  const {localDb, refresh, refreshWithId} = useLocalDatabaseHook();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->